deployments/porch/5-rbac.yaml

```diff
@@ -68,3 +68,6 @@ rules:
   - apiGroups: [""]
     resources: ["pods"]
     verbs: ["create", "delete", "patch", "get", "watch", "list"]
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["create", "delete", "update", "get"]
```

Comment:
they are required yes as the pod manager in the function runner needs to have this authority to 

- create a replica of the secret given by the user onto the porch-fn-system namespace
- remove it from the cluster when the fn-runner is itself removed
- get the secrets data to compare with the one given by the user for in case the user edits/updates that secret content with new data it should be propagated to the secret on the pods.
- and lastly when the secret content is updated by the user we must propagate that update to the replica secret.